docs/BreakingChanges/051 WPF TextBox defaults to undo limit of 100.md

```diff
@@ -0,0 +1,22 @@
+## 51: WPF TextBox defaults to undo limit of 100
```

Comment:
{
  "Id": "51",
  "Title": "WPF TextBox defaults to undo limit of 100",
  "Details": "In .NET 4.5, the default undo limit for a WPF textbox is 100 (as opposed to being unlimited in .NET 4.0)",
  "Suggestion": "If an undo limit of 100 is too low, the limit can be set explicitly with the TextBox's UndoLimit property",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "T:System.Windows.Controls.TextBox"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] T:System.Windows.Controls.TextBox